src/pages/cryptoPage.jsx

```diff
@@ -0,0 +1,56 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { getCryptoPrices, coinInfo } from '../redux/crypto/cryptoSlice';
+import CryptoInfo from './cryptoInfo';
+import CryptoImage from '../assets/CRYPTOunsplash.webp';
+
+const CryptoPage = () => {
+  const dispatch = useDispatch();
+  const cryptoArr = useSelector((state) => state.crypto.cryptoArr);
+  const loading = useSelector((state) => state.crypto.loading);
+
+  useEffect(() => {
+    if (cryptoArr.length === 0) {
+      dispatch(getCryptoPrices());
+    }
+  }, [dispatch, cryptoArr]);
+  const coinInfoHandel = (id) => {
+    dispatch(coinInfo(id));
+  };
+
+  if (loading) {
+    return <p>Loading...</p>;
+  }
+  if (cryptoArr.length > 1) {
+    return (
+      <>
+        <h2 className="crypto-h2">Crypto Prices in USD</h2>
+        <section>
+          <img src={CryptoImage} alt="" />
+        </section>
+        <div>
+          {cryptoArr ? (
+            <div className="crypto-container">
```

Comment:
- [ ] Please make sure every page (the main page and the pages for each item) should have a unique route as requested in the project requirements. Currently, All pages are showing on the same route(root route), For a unique route you can use the names of cryptos and pass them in the URL to use in the route. You can read more about React dynamic routing [here](https://dev.to/shubhamtiwari909/dynamic-routes-in-react-491g)

![image](https://user-images.githubusercontent.com/83803414/225131891-6660557e-5184-435e-8410-7aba3b72c4ae.png)